tests/trainer/test_dataloaders.py

```diff
@@ -251,6 +251,52 @@ def test_multiple_dataloaders_passed_to_fit(tmpdir, ckpt_path):
         f'Multiple `test_dataloaders` not initiated properly, got {trainer.test_dataloaders}'
 
 
+@pytest.mark.parametrize(
+    ['limit_train_batches', 'limit_val_batches', 'limit_test_batches'],
+    [
+        pytest.param(0.0, 0.0, 0.0),
```

Comment:
Since `limit_*_batches` can be an `int` now, I think this test should cover cases with integer values as well for `limit_*_batches`? Thoughts?